### PR TITLE
Revert to 2 params as isinstance cannot support 3+

### PR DESCRIPTION
**Pull request by kimihiro64**

### CODE REVIEW

File: lbrynet/lbrynet_daemon/LBRYDaemonCLI.py [hunk 1]
```diff
@@ -35,7 +35,7 @@ def main():
     if len(args) > 1:
         if isinstance(args[1], dict):
             params = args[1]
-        elif isinstance(args[1], str, unicode):
+        elif isinstance(args[1], str) or isinstance(args[1], unicode):
```
Comment (jobevers) [inline lbrynet/lbrynet_daemon/LBRYDaemonCLI.py]: `elif isinstance(args[1], basestring):`

Comment (jobevers) [inline lbrynet/lbrynet_daemon/LBRYDaemonCLI.py]: can also use a tuple for the classes: https://docs.python.org/2/library/functions.html#isinstance

**Comment by kimihiro64** [comment]
There we go.

**Comment by jobevers** [comment]
Travis has had difficulties on osx, merging despite the build failure there.

**Pull request merged**